cpp/Glacier2/callback/MockAlarmClock.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) ZeroC, Inc.
+
+#ifndef MOCK_ALARM_CLOCK_H
+#define MOCK_ALARM_CLOCK_H
+
+#include "AlarmClock.h"
+
+#include <future>
+
+namespace Client
+{
+    /// MockAlarmClock is an Ice servant that implements Slice interface AlarmClock.
```

Comment:
`@p AlarmClock`? We should not use autolink here.

Reply:
I'll open an issue. There are a ton of these in many files.